public/components/trace_analytics/components/common/plots/service_map.tsx

```diff
@@ -204,9 +243,27 @@ export function ServiceMap({
 
   const onFocus = (service: string, networkInstance?: any) => {
     if (service.length === 0) {
+      // Reset all nodes to the default size when no service is selected
+      const resetNodes = items.graph.nodes.map((node) => ({ ...node, size: 15 }));
+      setItems({
+        ...items,
+        graph: { ...items.graph, nodes: resetNodes },
+      });
+      if (networkInstance) networkInstance.fit(); // Adjust the view if needed
       setInvalid(false);
     } else if (serviceMap[service]) {
       if (!networkInstance) networkInstance = network;
+
+      // Enlarge the focused node and reset others
+      const updatedNodes = items.graph.nodes.map((node) =>
+        node.label === service ? { ...node, size: 30 } : { ...node, size: 15 }
```

Comment:
just wondering, is there some global or environmental variable that can be used for the size here to get rid of the magic numbers?